### PR TITLE
En_Ssh

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

Docs from OoT

### CODE REVIEW

File: src/overlays/actors/ovl_En_Ssh/z_en_ssh.c [hunk 1]
```diff
@@ -37,139 +37,900 @@ const ActorInit En_Ssh_InitVars = {
...
+
+        gSPDisplayList(POLY_OPA_DISP++, object_ssh_DL_0000D8);
+
+        CLOSE_DISPS(globalCtx->state.gfxCtx);
+    }
+    Collider_UpdateSpheres(limbIndex, &this->collider2);
+}
+
+void EnSsh_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    static TexturePtr D_80976178[] = { &object_ssh_Tex_001970, &object_ssh_Tex_001DF0, &object_ssh_Tex_0021F0 };
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Ssh/z_en_ssh.c:919]: I think you don't need the `&` there
```suggestion
    static TexturePtr D_80976178[] = { object_ssh_Tex_001970, object_ssh_Tex_001DF0, object_ssh_Tex_0021F0 };
```

**Review by AngheloAlf — COMMENTED** [review]
Nice work. Just a minor comment

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/actors/ovl_En_Ssh/z_en_ssh.c [hunk 2]
```diff
@@ -37,139 +37,900 @@ const ActorInit En_Ssh_InitVars = {
...
+
+    CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(2), &sColChkInfoInit);
+    Collider_InitJntSph(globalCtx, &this->collider2);
+    Collider_SetJntSph(globalCtx, &this->collider2, &this->actor, &sJntSphInit, this->collider2Elements);
+}
+
+f32 EnSsh_SetAnimation(EnSsh* this, s32 arg0) {
+    AnimationHeader* animation[] = { &object_ssh_Anim_006D78, &object_ssh_Anim_001494, &object_ssh_Anim_001494,
+                                     &object_ssh_Anim_006788, &object_ssh_Anim_001494, &object_ssh_Anim_001494,
+                                     &object_ssh_Anim_006D78 };
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Ssh/z_en_ssh.c:212]:
```suggestion
    AnimationHeader* animation[] = { &object_ssh_Anim_006D78, &object_ssh_Anim_001494, &object_ssh_Anim_001494,
                                     &object_ssh_Anim_006788, &object_ssh_Anim_001494, &object_ssh_Anim_001494,
                                     &object_ssh_Anim_006D78, };
```
Can you add the trailing comma and reformat?

File: src/overlays/actors/ovl_En_Ssh/z_en_ssh.c [hunk 3]
```diff
@@ -37,139 +37,900 @@ const ActorInit En_Ssh_InitVars = {
...
+    return EnSsh_CheckHitBack(this, globalCtx);
+}
+
+void EnSsh_SetBodyCylinderAC(EnSsh* this, GlobalContext* globalCtx) {
+    Collider_UpdateCylinder(&this->actor, &this->collider1[0]);
+    CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider1[0].base);
+}
+
+void EnSsh_SetLegsCylinderAC(EnSsh* this, GlobalContext* globalCtx) {
+    s16 angleTowardsLink = ABS_ALT((s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y));
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Ssh/z_en_ssh.c:572]:
```suggestion
    s16 angleTowardsLink = ABS_ALT(BINANG_SUB(this->actor.yawTowardsPlayer, this->actor.shape.rot.y));
```

**Review by hensldm — APPROVED** [review]
Couple cosmetic suggestions.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**